src/components/connector-authentication/useConnectorAuthentication.ts

```diff
@@ -1,81 +1,98 @@
 import { RefreshedAuthCredendentials } from '@chili-publish/studio-sdk';
-import { useCallback, useMemo, useState } from 'react';
+import { useCallback, useState } from 'react';
 import { ConnectorAuthenticationResult } from '../../types/ConnectorAuthenticationResult';
+import { useConnectorAuthenticationResult } from './useConnectorAuthenticationResult';
 
 interface Executor {
     handler: () => Promise<ConnectorAuthenticationResult>;
 }
 
+export type ConnectorAuthenticationFlow = {
```

Comment:
Can we separate authentication process management from authentication flows? I don't like idea of using `useConnectorAuthenticationResult();` in this hook